common/schemas.ts

```diff
@@ -1,11 +1,11 @@
 import { z } from "npm:zod";
-enum ProjectRoles {
+export enum ProjectRoles {
   CREATOR = "CREATOR",
   VIEWER = "VIEWER",
   REVIEWER = "REVIEWER",
 }
```

Comment:
Nit: Entirely a preference thing, but `enum`s are sort of weird in TypeScript as they are one of the few features that require extra compiler logic and thus aren't fully supported by all TypeScript tooling. Typically instead of an enum, you could use a union type:
```suggestion
export type ProjectRoles = "CREATOR" | "VIEWER" | "REVIEWER";
```